common/i18n/resources/en.js

```diff
@@ -455,32 +455,55 @@ module.exports = {
 
   authGroups: {
     systemAdmin: {
-      label: 'System Administrators',
+      label: 'System Administrator',
+      label_plural: 'System Administrators',
       description: 'OF Arena system administrators',
     },
     surveyAdmin: {
-      label: 'Survey administrators',
+      label: 'Survey administrator',
+      label_plural: 'Survey administrators',
       description: 'Full rights',
     },
     surveyEditor: {
-      label: 'Survey editors',
+      label: 'Survey editor',
+      label_plural: 'Survey editors',
       description: 'Can edit survey, records, invite users',
     },
     dataEditor: {
-      label: 'Data editors',
+      label: 'Survey editor',
+      label_plural: 'Data editors',
       description: 'Can edit records in data entry step',
     },
     dataCleanser: {
-      label: 'Data cleansers',
+      label: 'Data cleanser',
+      label_plural: 'Data cleansers',
       description: 'Can edit records in data cleansing step',
     },
     dataAnalyst: {
-      label: 'Data analysts',
+      label: 'Data analyst',
+      label_plural: 'Data analysts',
       description: 'Can edit records in data analysis step',
     },
     surveyGuest: {
       label: 'System guest',
+      label_plural: 'System guests',
       description: 'Can view records',
```

Comment:
surveyGuest label should be 'Survey guest' not 'System guest'.

Reply:
Right, changed